test/index.js

```diff
@@ -118,28 +112,20 @@ describe('DoubleClick Floodlight', function(){
     });
 
     it('should set copa compliance', function(done) {
-      var json = test.fixture('app-install');
+      var json = test.fixture('app-install-copa');
+      var expectedUrl = 'https://ad.doubleclick.net/ddm/activity/' + json.output.endpoints[0];
 
       // set integration option
       json.input.integrations = {
         'DoubleClick Floodlight': {
           copaCompliant: true
         }
       };
-      var output = json.output;
-      var expectedUrl = 'https://ad.doubleclick.net/ddm/activity/'
-        + 'dc_rdid=' + output.dc_rdid
-        + ';src=' + output.src
-        + ';cat=' + output.cat
-        + ';type=' + output.type
-        + ';ord=' + '2700503028455676400'
-        + ';tag_for_child_directed_treatment=1'
-        + ';dc_lat=' + output.dc_lat;
 
       test
         .track(json.input)
         .expects(200)
-        .set('User-Agent', output.userAgent)
+        .set('User-Agent', json.output.userAgent)
         .end(function(err, res){
           if (err) throw err;
           assert.strictEqual(res[0].request.url, expectedUrl);
```

Comment:
@tsholmes ahhh sad https://github.com/segmentio/integration-tester/blob/4fd6450bddab9f4c5e0e41a0cbb4144310d45b92/lib/index.js#L362

only checks the base endpoint, not the full req url made :(